core/src/main/java/org/infinispan/configuration/global/GlobalAuthorizationConfiguration.java

```diff
@@ -44,36 +44,47 @@ static AttributeSet attributeDefinitionSet() {
 
    static {
       DEFAULT_ROLES = Map.of(
-            "admin", Role.newRole("admin", true,
+            "admin", Role.newRole("admin",
+                  "Superuser with all permissions including control of the Cache Manager lifecycle",
```

Comment:
These strings should be internationalized and retrieved via the `Messages` class which currently lives in `org.infinispan.util.logging.events` but should be bumped up one level

Reply:
@domiborges please review the content of these strings

Reply:
this content is he content in the docs

Reply:
Done